src/higlass_schema/schema.py

```diff
@@ -26,10 +26,16 @@ class BaseModel(PydanticBaseModel):
     def __rich_repr__(self):
         return iter(self)
 
-    def model_dump(self, exclude_none: bool = True, **kwargs):
+    def dict(self, exclude_none: bool = True, **kwargs) -> Dict[str, Any]:
```

Comment:
We should avoid changing the behavior of `dict`/`json` even if they are deprecated. Users will get a warning to prefer the newer APIs.